ros2run/ros2run/api/__init__.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import subprocess
+
+from ros2cli.node.strategy import NodeStrategy
+from ros2pkg.api import get_prefix_path
+
+
+class MultipleExecutables(Exception):
+
+    def __init__(self, paths):
+        self.paths = paths
+
+
+def get_executable_paths(*, node, package_name):
+    prefix_path = get_prefix_path(package_name)
```

Comment:
Now it says `No executable found`, which to be more accurate, the package was not found. It would be better if it said that.

```
% ros2 run taco talker
No executable found
```